cpi/tests/functional.rs

```diff
@@ -5,22 +5,34 @@ use {
         rent::Rent,
         system_program,
     },
-    solana_program_rosetta_cpi::{process_instruction, SIZE},
+    solana_program_rosetta_cpi::SIZE,
     solana_program_test::*,
     solana_sdk::{account::Account, signature::Signer, transaction::Transaction},
     std::str::FromStr,
 };
 
+/// The name of the program to test when using the `solana_program` library.
+const SOLANA_PROGRAM: &str = "solana_program_rosetta_cpi";
+
+/// The name of the program to test when using the `pinocchio` library.
+const PINOCCHIO_PROGRAM: &str = "pinocchio_rosetta_cpi";
+
 #[tokio::test]
 async fn test_cross_program_invocation() {
     let program_id = Pubkey::from_str("invoker111111111111111111111111111111111111").unwrap();
     let (allocated_pubkey, bump_seed) =
         Pubkey::find_program_address(&[b"You pass butter"], &program_id);
+
+    let library = std::env::var("ROSETTA_LIBRARY").unwrap_or(String::from("solana_program"));
```

Comment:
Needing to specify the `ROSETTA_LIBRARY` env variable *and also* hard-code it into the test seems a bit overkill -- how about having an environment variable like `PROGRAM_NAME` and just using that or the default?

Then `test-pinocchio.sh` just needs to combine `pinocchio` with the program name. What do you think?

Reply:
Hmmm, but we still have the issue that `ProgramTest::new` expects the `program_name` as a `&'static str`. That is the only reason for hard-coding the name on the test file.

Reply:
how about using `env!` in that case? https://doc.rust-lang.org/std/macro.env.html

Reply:
Great idea - ended up using `option_env!` so we can specify a default value if the variable is not set.